build.teamcity.ps1

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+$ErrorActionPreference = 'Stop'
+
+& "$PSScriptRoot/Initialize-PowershellForDevelopment.ps1"
+
+$teamcityParameters = Get-TeamCityParameters
+Write-Host
+Write-Host "$($teamcityParameters.Count) TeamCity parameters found."
+Write-Host
+
+# Build and Test
+$params = @{
+    InstallType   = Get-ValueOrDefault                    $teamcityParameters['odsapi.build.installType']    'Sandbox'
+    OdsTokens     = Get-ValueOrDefault (ConvertTo-Array   $teamcityParameters['odsapi.build.odsTokens'])     @()
+    Engine        = Get-ValueOrDefault                    $teamcityParameters['odsapi.build.engine']         'SQLServer'
+    NoCodeGen     = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.noCodeGen'])     $false
+    NoRebuild     = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.noRebuild'])     $false
+    NoDeploy      = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.noDeploy'])      $false
+    RunPester     = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runPester'])     $true
+    RunDotnetTest = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runDotnetTest']) $true
+    RunPostman    = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runPostman'])    $true
+    RunSmokeTest  = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.runSmokeTest'])  $true
+    UsePlugins    = Get-ValueOrDefault (ConvertTo-Boolean $teamcityParameters['odsapi.build.usePlugins'])    $false
+}
+
+Invoke-Task "Remove-EdFiDatabases" { Remove-EdFiDatabases -Force -Engine $params.Engine }
```

Comment:
Should this be an option to run as part of initdev as well?  This might actually be nice to use locally...  I guess you could always just call it explicitly for now.

Reply:
yeah I think that would be useful at some point. I held off here because to might be a little more destructive than some may like.